product_docs/docs/epas/16/reference/oracle_compatibility_reference/epas_compat_cat_views/pg_views/pg_class.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: "PG_CLASS"
+---
+
+The `PG_CLASS` view provides information about tables and other objects that have columns or in other ways are similar to a table. This includes indexes, sequences, views, and other objects. In the context of this table "objects" refer to "relations." In addition to the information in the [PostgreSQL pg_class description](https://www.postgresql.org/docs/current/catalog-pg-class.html), EDB Postgres Advanced Server provides the following information:
+
+| Name                  | Type    | Description                                                                                           |
+| --------------------- | ------- | ----------------------------------------------------------------------------------------------------- |
+| relbackingtable       | "char"  | Specifies whether a table is backing relation for any other object. The default value is `d` for a normal object and `q` for queue tables. We store the relbackingtable as `q` for all the backing objects of QUEUE TABLE (backing table, index, toast table, and so forth). |
```

Comment:
Using relbackingtable we can identify whether table is backing relation for
 * any other object. Default value for relbackingtable will be 'd' for normal
 * object and 'q' for queue tables. We store the relbackingtable as 'q' for all
 * the backing object of QUEUE TABLE (backing table, index, toast table, etc).
 */